boot_default/config.txt

```diff
@@ -73,3 +73,6 @@ dtparam=audio=on
 
 # for more options see http://elinux.org/RPi_config.txt
 
+[pi3]
+# for light board
+enable_uart=1
```

Comment:
My only concern is that potentially we might append to this file, in which case the appended statements would only apply to RPi 3. Maybe it would be wise to finalise with a `[all]`?

More info at the bottom of the [config.txt docs](https://www.raspberrypi.org/documentation/configuration/config-txt.md).